rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -429,40 +404,19 @@ object Runtime {
     }
   }
 
-  def injectEmptyRegistryRoot[F[_]](space: RhoISpace[F], replaySpace: RhoReplayISpace[F])(
-      implicit F: Concurrent[F]
-  ): F[Unit] = {
-    // This random value stays dead in the tuplespace, so we can have some fun.
-    // This is from Jeremy Bentham's "Defence of Usury"
-    val rand = Blake2b512Random(
-      ("there can be no such thing as usury: " +
-        "for what rate of interest is there that can naturally be more proper than another?")
+  def bootstrapRegistry[F[_]: FlatMap](runtime: Runtime[F]): F[Unit] = {
+    // This is from Nassim Taleb's "Skin in the Game"
+    implicit val rand = Blake2b512Random(
```

Comment:
@ArturGajowy and I were chatting about having a new quote for the rholang-based registry.
This is my suggestion. Happy to take other suggestions or we can take a vote or something :)

Reply:
How about:
> I only take advice from a utilitarian when it maximizes happiness and well-being for the majority of a population.  -Chris Boscolo

Reply:
Quote taken aside, I love the act itself. It's a great way to stay true to the original implementation's spirit :joy: